### DISCUSSION TITLE
Suggestion: dark mode setting should be saved with patch

### DISCUSSION THREAD

**Discussion by GBaige** [DISTRHO/Cardinal#523] [Ideas]
The current behavior is that Cardinal will load with dark mode applied by default. If a user wishes to use light mode, they'll have to disable dark mode manually every time they load a new instance of Cardinal in a separate project (since Cardinal settings are saved per project).

A few days ago I saved a patch containing some Surge XT modules. At the time, I had dark mode disabled. Today I loaded the same patch in another project, and every other module but the Surge XT modules followed the dark mode setting. For this to occur is very rare but it does happen. Closing and opening the DAW fixes the mismatch, and the same patch will no longer load with Surge XT modules in light mode.

![image](https://user-images.githubusercontent.com/115324169/230534464-edb2efbc-246f-466f-94d7-e45d7315a4e4.png)

This gave me an idea, though: what if the dark mode setting (or any other setting for that matter) was saved with the patch? Is there a technical limitation or reason as to why this wouldn't be possible?

On a side note, wouldn't it be possible for Cardinal to remember settings (perhaps via a separate config file within the resources folder, or next to the .dll/.vst3/etc.)? A workaround to this is to create a template for the DAW where Cardinal has been pre-configured, but I wouldn't want to have to do that if I used multiple DAWs.

**Comment by GBaige** [comment]
I guess very few people use Cardinal to actually care about stuff like this.

**Reply by falkTX** [reply]
or it needs more hands, you are always welcome to contribute the feature yourself.
